.github/workflows/publish.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish
```

Comment:
This workflow is not applicable here. We have another one called `publish-gcloud.yml` instead.